koku/api/settings/settings.py

```diff
@@ -228,6 +268,35 @@ def _tag_key_handler(self, settings):
 
         return any(updated)
 
+    def _currency_handler(self, settings):
+        currency = settings
+        try:
+            stored_currency = get_selected_currency_or_setup(self.schema)
+        except Exception as exp:
+            LOG.warning(f"Failed to retrieve currency for schema {self.schema}. Reason: {exp}")
+            return False
```

Comment:
Is the log output to the browser console? If so, it's not desirable for production.

Reply:
Would only be logged to the pod logs in OpenShift